docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Use a custom domain to expose a service
+---
+
+This tutorial shows how to set up your custom domain and prepare a certificate to use the domain for exposing a service. The components used are Gardener [External DNS Management](https://gardener.cloud/docs/concepts/networking/dns-managment/#external-dns-management) and [Certificate Management](https://gardener.cloud/docs/concepts/networking/cert-managment/).
+
+To learn how to expose a service, go to [this](./apix-01-expose-service-apigateway.md) tutorial.
+
+## Prerequisites
+
+If you use a cluster not managed by Gardener, install the required components manually. Follow these steps:
+
+1. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+2. Download the [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) projects locally.
+
+3. Enable vertical Pod autoscaling on your cluster. For example, for Google Cloud Platform, run:
+
+   ```bash
+   gcloud beta container clusters update {PROJECT_NAME} --enable-vertical-pod-autoscaling
+   ```
+
+4. Install the `external-dns-management` component in your Namespace:
+
+   ```bash
+   helm install external-dns {PATH_TO_EXTERNAL_DNS_MANAGEMENT}/charts/external-dns-management --namespace={NAMESPACE_NAME} --set configuration.identifier=external-dns-identifier
+   ```
+
+5. Install the `cert-management` component in your Namespace:
+
+   ```bash
+   helm install cert-manager {PATH_TO_CERT_MANAGEMENT}/charts/cert-management --namespace={NAMESPACE_NAME} --set configuration.identifier=cert-manager-identifier
+   ```
+
+When you meet the prerequisites, go directly to step 2.
+
+## Steps
+
+Follow these steps to set up your custom domain and prepare a certificate required to expose a service.
+
+1. Install Kyma 2.0 or higher.
+
+2. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
```

Comment:
do we need double namespace creation?

Reply:
No, we don't. Maybe it should be stated verbosely: `Perform this step if not done as a part of prerequisites.`